live-examples/html-examples/embedded-content/source.html

```diff
@@ -0,0 +1,12 @@
+<video controls
+    width="300"
+    height="200"
+    autoplay="true"
+    loop="true"
```

Comment:
I think we should omit the `autoplay` and `loop` attributes. I appreciate that you want to demo them, but I think it would be distracting for people who don't want to interact with the example, and are just visiting the page to look something up (we could omit just `autoplay`, but I'd prefer to omit both I think).